packages/components/src/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -220,7 +255,11 @@ export const ConfirmationModal = forwardRef(function ConfirmationModalInternal(
       }}
     >
       <Content>
-        <Text>{state.message}</Text>
+        <Markdown
+          content={state.message}
+          externalLink={state.externalLink}
+          basicUsage={state.basicUsage}
```

Comment:
I think that these props are out of scope for https://github.com/GetJobber/atlantis/issues/381. In the issue, the only change should be that the message prop can accept markdown.

Reply:
The issue requested using the Markdown component, should it not be possible to pass these props to the markdown component?

Reply:
Removed!